site/en/blog/webauthn-conditional-ui/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Passwordless sign-in on forms with WebAuthn passkey autofill
+subhead: WebAuthn conditional UI leverages browser's form autofill functionality to let users sign in with a passkey seamlessly in the traditional password based flow.
+authors:
+  - agektmr
+date: 2022-11-17
+hero: image/YLflGBAPWecgtKJLqCJHSzHqe2J2/xMIoePcEqw5uHCkxY3nP.jpg
+alt: A man filling a form
+description: WebAuthn conditional UI leverages browser's form autofill functionality to let users sign in with a passkey seamlessly in the traditional password based flow.
+tags:
+  - webauthn
+  - identity
+  - security
+  - passkeys
+---
+
+Chrome 108 supports passkeys, including autofill suggestions. This allows sites
+to build easy sign-in experiences that are more secure.
+
+## What is a passkey?
+
+A [passkey](https://developers.google.com/identity/passkeys) is a digital credential, tied to a user account and a website or application. Passkeys are more secure than
+passwords because they enable users to sign in to apps and websites with a
+biometric sensor (such as a fingerprint or facial recognition), PIN, or pattern,
+freeing them from having to remember and manage passwords. Because they are
+[WebAuthn](https://www.w3.org/TR/webauthn-2/) credentials, they include the
+following security features:
+
+* Passkeys use public-key cryptography so sites don't have to store passwords or secrets.
+* Users can't be phished because a passkey can only be used with the website or app that created it.
+
+Websites can create and use WebAuthn credentials on devices and
+security keys via a modal UI in the browser. To do so, websites have to
```

Comment:
```suggestion
Websites could create and use WebAuthn credentials on devices and
security keys via a modal UI in the browser. To do so, websites had to
```

Reply:
@anusmitaray This was the past status. By adding the new conditional UI, the UX is seamless.

Reply:
Okay that makes sense to change both to the past tense then.